flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/format/single/SingleValueRowSerializerTest.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.format.single;
+
+import org.apache.flink.table.data.GenericRowData;
+import org.apache.flink.table.data.RowData;
+import org.apache.flink.table.data.RowData.FieldGetter;
+import org.apache.flink.table.data.StringData;
+import org.apache.flink.table.runtime.typeutils.RowDataTypeInfo;
+import org.apache.flink.table.types.logical.BigIntType;
+import org.apache.flink.table.types.logical.BooleanType;
+import org.apache.flink.table.types.logical.CharType;
+import org.apache.flink.table.types.logical.DoubleType;
+import org.apache.flink.table.types.logical.FloatType;
+import org.apache.flink.table.types.logical.IntType;
+import org.apache.flink.table.types.logical.LogicalType;
+import org.apache.flink.table.types.logical.RowType;
+import org.apache.flink.table.types.logical.SmallIntType;
+import org.apache.flink.table.types.logical.TinyIntType;
+import org.apache.flink.table.types.logical.VarBinaryType;
+import org.apache.flink.table.types.logical.VarCharType;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests for {@link SingleValueRowDataDeserialization} {@link SingleValueRowDataSerialization}.
+ */
+public class SingleValueRowSerializerTest {
+
+	@Test
+	public void testDeSeSingleValue() throws IOException {
+		String testString = "hello,world";
+		byte[] testBytes = testString.getBytes();
+		int testInt = 1024;
+		short testShort = 1024;
+		byte testByte = (byte) 1;
+		float testFloat = 1024.0f;
+		double testDouble = 1024.0d;
+		long testLong = 1024L;
+		boolean testBoolean = false;
+		char testChar = 'T';
+
+		testSeDeSingleValue(getValueBytes(testString), StringData.fromString(testString), new VarCharType());
```

Comment:
I think we can use the serializer to convert the Object into byte[], and use the deserializer to convert byte[] back to Object, and compare the result Object with the original Object. In this way, we don't need the `getValueBytes`.